Makefile

```diff
@@ -51,11 +51,21 @@ EXT_JERRY_FLAGS ?=	-DENABLE_ALL_IN_ONE=ON \
 					-DFEATURE_PROFILE=$(ZJS_BASE)/jerry_feature.profile \
 					-DFEATURE_ERROR_MESSAGES=OFF \
 					-DJERRY_LIBM=OFF
-ifneq ($(DEV), ashell)
+# Settings for ashell builds
+ifneq (,$(filter $(MAKECMDGOALS),ide ashell))
+SNAPSHOT=off
+# if the user passes in SNAPSHOT=on give an error
 ifeq ($(SNAPSHOT), on)
-EXT_JERRY_FLAGS += -DFEATURE_JS_PARSER=OFF
+$(error IDE does not support SNAPSHOT $(SNAPSHOT))
```

Comment:
It needs to be an error, if the user passes in SNAPSHOT=on I can't turn it off.  Which means it won't include the parser, which means ashell will fail.